### ISSUE TITLE
FATAL_ERROR task state

### ISSUE THREAD

**Issue by lithorus** [CGRU/cgru#602]
We're tracking resource usage (eg. memory), through the service and parser and will fail the task if it goes above the limit set (we're currently using capacity for that). However it will just error out and retry again according to the number retries allowed.

So my suggestion would be to add a new task state (eg. FATAL_ERROR) that will not retry the task unless it's un-blocked by the user. There might also be other use cases where you don't want to retry it after it fails.

**Comment by lithorus** [comment]
Any thoughts about this?

**Comment by timurhai** [comment]
Yes, it can be useful. And it is not hard to implement.

**Comment by sebastianelsner** [comment]
Yes, I see this can be useful as well. We have a similar system as you @lithorus but we do not have retries set per task so the task is blocked/stopped immediately.

**Comment by lithorus** [comment]
I've started to look at it and might create a MR with the changes.

**Comment by timurhai** [comment]
So, how is going?
Now it is the only uncompleted issue of a 3.4.0 milestone.
@lithorus

**Comment by lithorus** [comment]
That would be great, I won't have much time to look at it myself.
